src/Microsoft.Diagnostics.Monitoring.WebApi/Exceptions/ExceptionsSettingsFactory.cs

```diff
@@ -19,12 +19,12 @@ public static ExceptionsConfigurationSettings ConvertExceptionsConfiguration(Exc
                 return configurationSettings;
             }
 
-            foreach (var filter in configuration.Include)
+            foreach (var filter in configuration.Include ?? [])
```

Comment:
Could you null check instead of creating an array and enumerating an empty array? Same on line 27.